agent/taskresource/cgroup/cgroup.go

```diff
@@ -331,3 +331,13 @@ func (cgroup *CgroupResource) UnmarshalJSON(b []byte) error {
 	}
 	return nil
 }
+
+// GetCgroupRoot returns cgroup root of the resource
+func (cgroup *CgroupResource) GetCgroupRoot() string {
+	return cgroup.cgroupRoot
+}
+
+// GetCgroupMountPath returns cgroup mount path of the resource
+func (cgroup *CgroupResource) GetCgroupMountPath() string {
+	return cgroup.cgroupMountPath
```

Comment:
these are static fields right? set during agent init?

Reply:
these are set from the config values.